setup.py

```diff
@@ -29,6 +29,7 @@ def read_version():
 required_gpsearchers = load_requirements('requirements-gpsearchers.txt')
 required_bore = load_requirements('requirements-bore.txt')
 required_blackbox_repository = load_requirements('blackbox_repository/requirements.txt')
+required_kde = load_requirements('requirements-kde.txt')
```

Comment:
This is almost getting a bit fine-grained. One could also lump this together with gpsearchers

Reply:
I would keep it like this for now, but in the future we could think of having something like 'requirements-models.txt' that bundles model-based optimizers together?

Reply:
OK